### ISSUE TITLE
build-apidocs script should use the pyOpenSSL and Python intersphinxes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8290]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8290|
|Type|enhancement|
|Created|2016-04-13 15:25:32Z|
|Branch|https://github.com/twisted/twisted/tree/apidocs-intersphinx-8290|

This will mean we link to the docs of these libraries nicely.

<details><summary>Searchable metadata</summary>

```
trac-id__8290 8290
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__apidocs_intersphinx_8290 apidocs-intersphinx-8290
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1460561132529567 1460561132529567
changetime__1464495403687635 1464495403687635
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [47221]) Branching to apidocs-intersphinx-8290.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

I'm happy with this.

For reviewers, compare the errors on trunk: https://buildbot.twistedmatrix.com/builders/documentation/builds/6574/steps/api-documentation/logs/pydoctor%20errors with this branch: https://buildbot.twistedmatrix.com/builders/documentation/builds/6582/steps/api-documentation/logs/pydoctor%20errors .

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Is this working?

I can still see errors to SSL.Context for example from here 

https://github.com/twisted/twisted/blob/d2eb6ff8a032fbc8cbc3988009234381bce8526e/twisted/internet/_sslverify.py#L1172

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl removed owner|
|-|-|

I cleaned that up a bit; please review.

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_6"></a>|@Lukasa set owner to @hawkowl|
|-|-|

LGTM. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 3455a902fb15e732ee43b59f4d82a66b105351ba
```
#!CommitTicketReference repository="" revision="3455a902fb15e732ee43b59f4d82a66b105351ba"
Merge apidocs-intersphinx-8290: Make the API docs use intersphinx to create links.

Author: hawkowl
Reviewers: adiroiban, lukasa
Fixes: #8290
```
